rasa/constants.py

```diff
@@ -49,9 +49,14 @@
 
 LEGACY_DOCS_BASE_URL = "http://legacy-docs.rasa.com"
 
-CONFIG_MANDATORY_KEYS_CORE = ["policies"]
-CONFIG_MANDATORY_KEYS_NLU = ["language", "pipeline"]
+CONFIG_KEYS_CORE = ["policies"]
+CONFIG_KEYS_NLU = ["language", "pipeline"]
```

Comment:
It's just as easy to read as a string, and we're unlikely to change how they're called. It would probably be a pain to find all occurrences in the code base, so let's rather make them constants only if we ever decide to rename them.